Lib/fontbakery/profiles/googlefonts.py

```diff
@@ -4631,6 +4632,55 @@ def com_google_fonts_check_varfont_instance_names(ttFont):
     yield PASS, "Instance names are correct"
 
 
+@check(
+  id = 'com.google.fonts/check/varfont_duplicate_instance_names',
+  rationale = """
+    According to Google Font’s specs (as of 2020-06-26), only weight and italic tokens are allowed in instance names (= "ExtraBold Italic").
+
+    This variable font contains duplicate instance names, suggesting that instances are defined for more then one axis.
+
+    Instead, only a base set of fonts for the most general representation of the family can be defined through instances in the `fvar` table,
+    all other instances will have to be left to access through the `STAT` table.
+  """,
+  conditions = ['is_variable_font'],
+)
+def com_google_fonts_check_varfont_duplicate_instance_names(ttFont):
+  """Check variable font instances don't have duplicate names"""
+  # This check and the fontbakery.parse module used to be more complicated.
+  # On 2020-06-26, we decided to only allow Thin-Black + Italic instances.
+  # If we decide to add more particles to instance names, It's worthwhile
+  # revisiting our previous implementation which can be found in commits
+  # earlier than or equal to ca71d787eb2b8b5a9b111884080dde5d45f5579f
+  from fontbakery.constants import SHOW_GF_DOCS_MSG
+
+  seen = []
+  duplicate = []
+
+  for instance in ttFont['fvar'].instances:
+    name = ttFont['name'].getName(
+      instance.subfamilyNameID,
+      PlatformID.WINDOWS,
+      WindowsEncodingID.UNICODE_BMP,
+      WindowsLanguageID.ENGLISH_USA
+    ).toUnicode()
+    
+    if name in seen:
+      duplicate.append(name)
+
+    if not name in seen:
+      seen.append(name)
+
+  if duplicate:
+    duplicate_instances = "\n\t- ".join([""] + duplicate)
+    yield FAIL,\
+          Message('duplicate-instance-names',
+                  f'Following instances names are duplicate: {duplicate_instances}\n'
+                  f'\n'
+                  f'{SHOW_GF_DOCS_MSG}#fvar-instances')
```

Comment:
Please remove this line (and its preceding `\n`) since it will result in the following being printed in the log message:

```
Further info can be found in our spec  https://github.com/googlefonts/gf-docs/tree/master/Spec#fvar-instances
```

and there's not really any further info in there that is relevant to the user in this case. It is just a matter of duplicate names.